cloudflare/resource_cloudflare_argo_tunnel.go

```diff
@@ -97,6 +113,7 @@ func resourceCloudflareArgoTunnelImport(d *schema.ResourceData, meta interface{}
 	}
 
 	d.Set("name", tunnel.Name)
+	d.Set("cname", fmt.Sprintf("%s.argotunnel.com", tunnel.ID))
```

Comment:
as this is happening in the `Read` and we're calling `resourceCloudflareArgoTunnelRead` at the end of this method, we don't really need this twice.